src/flow.ts

```diff
@@ -332,6 +332,18 @@ export class Flow {
     return branch;
   }
 
+  /** Fork this flow to a child flow in branch */
+  forkTrueBranch(condExpr: ExpressionRef, resetBreakContext: bool = false): Flow {
+    let branch = this.fork(resetBreakContext);
+    branch.inheritNarrowedTypeIfTrue(condExpr);
+    return branch;
```

Comment:
If `inheritNarrowedTypeIfTrue` refactor to return `this` (Flow) it could be simplified to:
```ts
  forkTrueBranch(condExpr: ExpressionRef, resetBreakContext: bool = false): Flow {
    return this.fork(resetBreakContext).inheritNarrowedTypeIfTrue(condExpr);
 }
```

Reply:
I think it will reduce the readability. Why inherit conditional return a flow? It will cause additional attentions to find what return flow means.

Reply:
I see. ok in this case keep this as is